agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml

```diff
@@ -17,6 +17,7 @@ rules:
   - resourcequotas
   - persistentvolumes
   - persistentvolumeclaims
+  - endpoints
```

Comment:
This change was already made in 74cf9902. Can you remove this one? We still need to change the openshift one.

Reply:
Done